src/Featurizers/Components/Details/PipelineExecutionEstimatorImpl_details.h

```diff
@@ -865,7 +865,7 @@ class TransformerChainElement<
     }
 
     TransformerChainElement(Archive &ar) :
-        _pTransformer(std::make_unique<typename ThisEstimator::TransformerType>(ar)) {
+        _pTransformer(new typename ThisEstimator::TransformerType(ar)) {

```

Comment:
Now that we are using a bunch of `new` instead of `std::make_unique` will we need to have a bunch of deletes as well? Or because it is moving will it still be ok?

Reply:
It is still being assigned to an unique_ptr, the change is that the make_unique function isn't available.